pkg/storage/tpr/install_types.go

```diff
@@ -100,8 +105,12 @@ func (i *Installer) InstallTypes() error {
 
 	if allErrMsg != "" {
 		glog.Errorf("Failed to create Third Party Resource:\n%s)", allErrMsg)
-		return fmt.Errorf("Failed to create Third Party Resource:\n%s)", allErrMsg)
+		return ErrTPRInstall{fmt.Sprintf("Failed to create Third Party Resource:\n%s)", allErrMsg)}
```

Comment:
Will it make sense to keep the newline because the error message that comes back from the installer might contain multiline, each line for one failed resource and the reason it failed to install, so this is a list of error for install failures